site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -1603,7 +1603,6 @@ private String buildContextForTemplate(String preIndex, int index, VelocityPortl
 			}
 
 			if (serverConfigurationService.getBoolean("sitesetup.show.unpublished", false) && !securityService.isSuperUser()) {
-				views.put(SiteConstants.SITE_ACTIVE, rb.getString("java.myActive"));
```

Comment:
Shouldn't we remove the orphaned translation key from all the properties?

Reply:
Yes, agree